riotfile.py

```diff
@@ -916,6 +916,21 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
             pys=select_pys(min_version="3.6"),
             command="pytest {cmdargs} tests/contrib/asgi",
         ),
+        Venv(
+            name="mariadb",
+            command="python -m pytest {cmdargs} tests/contrib/mariadb",
+            venvs=[
+                Venv(
+                    pys=select_pys(min_version="3.6"),
+                    pkgs={
+                        "mariadb": [
+                            ">=1.0.0,<1.1.0",
```

Comment:
```suggestion
                            "~=0.9",  # Most recent `0.x.x` release
                            "~=1.0.0",  # Most recent `1.0.x` release
                            "~=1.0",  # Most recent `1.x.x` release
```

Reply:
I now see that `0.9.x` is listed as beta 🤔 what kind of effort is it to support `0.9.x` if it is super easy/basically the same support, then maybe it is worth including... otherwise we can add support later if requested

Reply:
Tried this but it looks like there are setup issues. Going to not include for now.:  

ERROR: Command errored out with exit status 2:
     command: /root/project/.riot/venv_py387/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-6un8vj37/mariadb_43af4698120740069f3c51294b346354/setup.py'"'"'; __file__='"'"'/tmp/pip-install-6un8vj37/mariadb_43af4698120740069f3c51294b346354/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-s3b3ojzd
         cwd: /tmp/pip-install-6un8vj37/mariadb_43af4698120740069f3c51294b346354/
    Complete output (1 lines):
    MariaDB Connector/Python requires MariaDB Connector/C >= 3.1.3, found version 3.1.13